### PR TITLE
📓Update story ad docs

### PR DESCRIPTION
**Pull request by calebcordry**

### CODE REVIEW

File: pages/content/amp-dev/documentation/guides-and-tutorials/develop/story_ads_best_practices.md [hunk 1]
```diff
@@ -65,7 +65,8 @@ Meta tag data specifies that the ad meets the AMP Story format, sets the CTA but
 </html>
 [/sourcecode]
 
-The `amp-cta-type` tag must include one of the [available options](#call-to-action-button-text-enum) for the CTA Button text enum. This ensures a consistent user experience for AMP Story readers.
+The `amp-cta-type` tag can be chosen from one of the [available options](#call-to-action-button-text-enum) for the CTA Button text value. This is recommended, as AMP will then automatically localize the text for you when appropriate.
+Alternatively, if you wish to handle localization for yourself, you can include your own custom text for this value.
```
Comment (CrystalOnScript) [inline pages/content/amp-dev/documentation/guides-and-tutorials/develop/story_ads_best_practices.md]: Nit, maybe update to:

Choosing the `amp-cta-type` tag from the available CTA Button text options is recommended. AMP will automatically localize predefined options when appropriate. 

Custom text is allowed, but you will need to implement your own localization.

File: pages/content/amp-dev/documentation/guides-and-tutorials/develop/story_ads_best_practices.md [hunk 2]
```diff
@@ -260,6 +261,22 @@ There are various tools you can use to encode videos and adjust the quality of t
 
 Ensure the size of your HLS segments are typically no more than 10 seconds in duration.
 
+## Animation
+If you wish to include animation in your story, there are a few things to be aware of. The concept of what is "visible" in an amp-story can be more complicated than other formats.
```
Comment (CrystalOnScript) [inline pages/content/amp-dev/documentation/guides-and-tutorials/develop/story_ads_best_practices.md]: Another nit, I'd combine the first two sentences to:

Animations have a few caveats in stories, such as the concept of what is "visible".

File: pages/content/amp-dev/documentation/guides-and-tutorials/develop/story_ads_best_practices.md (hunk 1 above)
Comment (calebcordry) [reply pages/content/amp-dev/documentation/guides-and-tutorials/develop/story_ads_best_practices.md]: Done.

File: pages/content/amp-dev/documentation/guides-and-tutorials/develop/story_ads_best_practices.md (hunk 2 above)
Comment (calebcordry) [reply pages/content/amp-dev/documentation/guides-and-tutorials/develop/story_ads_best_practices.md]: Done.

**Comment by calebcordry** [comment]
Thanks @CrystalOnScript !

**Pull request merged**